content/pages/contributing.md

```diff
@@ -0,0 +1,432 @@
+
+You can now make changes to your local copy of the Pythia repo
+without having those changes affect either the remote Pythia GitHub
+repo, your remote, personal GitHub repo, or your local repo in .git,
+until you are ready to merge your local changes upstream, first to
+your .git local repo, then to your remote GitHub repo, and then
+ultimately to the remote Pythia GitHub repo. Simple, right? More
+on this later.
+
+For further information see the [GitHub docs on forking a repo](https://docs.github.com/en/github/getting-started-with-github/fork-a-repo).
+
+Remember: at this point you should have a local copy of the repository
+in your current working directory. You can safely make changes to
+any of the contents. Once you are ready to contribute your changes
+back to the Pythia repository you will need to submit a Pull Request
+(PR), described later.
+
+### Make your local changes
+
+At this point you should be able to make your desired changes to
+the local copies of your files. Consult the repo-specific contributor’s
+guide for information specific to the repo you are working on.
+
+### Submit a Pull Request (PR)
+
+Once you have completed making changes to your local copy of a
+Pythia repository and are ready to have your changes merged with a
+Pythia repository on GitHub, you need to essentially perform the
+reverse processed used to acquire a copy of the Pythia repo, and
+submit a PR asking the Pythia maintainers to consider your merge
+request. The merge will occur between your personal GitHub repository
+and the Pythia GitHub repository, so you first need to merge any
+changes you’ve made in your local copy into your local .git repo.
+Next, you need to merge these local changes with your personal
+remote repo on GitHub. Finally, you need to submit a request to
+merge your personal GitHub repo with the Pythia GitHub repo.
+
+Git has lots and lots of commands, each with lots and lots of
+options. Here we only cover the very basics. Detailed information
+about Git can be found [here](https://git-scm.com/), but your best
+friend for figuring out to do things with Git may be Google, and
+in particular [StackOverflow](https://stackoverflow.com/).
+
+#### Committing Your Code Locally with Git
+
+Changes you’ve made to your local copy of a repository must be
+“committed” (merged) to your local repository (the .git subdirectory)
+using Git. You can see any uncommitted changes you’ve made to your
+local copy of the repository by running the following command from
+anywhere (any directory) within the directory where you ran git
+checkout:
+
+```
+$ git status
+```
+
+If you have added any new files you will need to explicitly add
+them to the local repo with:
+
+```
+$ git add PATH_TO_NEW_FILE
+```
+
+Where `PATH_TO_NEW_FILE` is the path name of the newly created file.
+
+To commit changed files, including new files just added with the above command, run the following command from the root of your local copy:
+
+```
+$ git commit
```

Comment:
Do we want to discuss the use of the precommit hooks here?

Reply:
What information do you think should be added?

Reply:
I've added a few sentences about pre-commit hooks. CC'ing @andersy005 and asking to re-review (this topic)

Reply:
See my comment below, I think we need to spell out the details: git add, git commit, let the hooks make changes, then git add, git commit again.